src/edu/psu/sweng500/team8/coreDataStructures/Cell.java

```diff
@@ -32,10 +32,10 @@ public Cell(int rowIndex, int columnIndex) {
 	}
 	
 	public Cell(Cell cellToCopy) {
-		m_coordinates = cellToCopy.m_coordinates;
+		m_coordinates = new CellCoordinates(cellToCopy.m_coordinates.getRowIndex(), cellToCopy.m_coordinates.getColumnIndex());

```

Comment:
It would be cleaner to create a copy constructor for the CellCoordinates class.

Was this a necessary change? Even if it shares the same coordinates object as another cell, it's immutable, so it shouldn't matter.